### PR TITLE
DBZ-2238 Fix test failure - MySqlSourceTypeInSchemaIT#shouldPropagateSourceTypeAsSchemaParameter

### PR DESCRIPTION
**Pull request by Naros**
https://issues.redhat.com/browse/DBZ-2238

### CODE REVIEW

File: debezium-connector-mysql/src/test/java/io/debezium/connector/mysql/MySqlSourceTypeInSchemaIT.java [hunk 1]
```diff
@@ -252,4 +252,30 @@ public void shouldPropagateSourceTypeByDatatype() throws SQLException, Interrupt
...
+
+        final StringBuilder result = new StringBuilder();
+        for (int i = 0; i < columns.length; ++i) {
+            final String column = columns[i];
+            result.append(DATABASE.qualifiedTableName(tableName) + "." + column);
+            if ((i + 1) < columns.length) {
+                result.append(",");
+            }
+        }
+        System.out.println(result.toString());
```
Comment (jpechane) [inline debezium-connector-mysql/src/test/java/io/debezium/connector/mysql/MySqlSourceTypeInSchemaIT.java:278]: I think this should be removed

**Comment by jpechane** [comment]
@Naros Nice catch! One question, would not be the easiest fix to modify the regexs to look like `.*\.c1` to include the dot in the pattern?

File: debezium-connector-mysql/src/test/java/io/debezium/connector/mysql/MySqlSourceTypeInSchemaIT.java (hunk 1 above)
Comment (Naros) [reply debezium-connector-mysql/src/test/java/io/debezium/connector/mysql/MySqlSourceTypeInSchemaIT.java:278]: Oh, I thought I had removed that before pushing.  Great catch.

**Comment by Naros** [comment]
> would not be the easiest fix to modify the regexs

Actually it would have and since I need to remove the system-out debugging, I'll go that route.  For whatever reason that didn't dawn on me when I saw what was creating the random failures.

**Comment by Naros** [comment]
@jpechane, I applied your advice and cleaned up the changes.  Thanks for spotting that!

**Comment by jpechane** [comment]
@Naros Applied, thanks!

**Pull request merged**